boxoffice/views/admin_order.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+
+from flask import jsonify, url_for
+from .. import app, lastuser
+from coaster.views import load_models, render_with
+from boxoffice.models import ItemCollection, CURRENCY_SYMBOL, ORDER_STATUS
+from utils import invoice_date_filter
+
+
+def format_assignee(assignee):
+    if assignee:
+        return {
+            'id': assignee.id,
+            'fullname': assignee.fullname,
+            'email': assignee.email,
+            'phone': assignee.phone,
+            'details': assignee.details
+        }
+
+
+def format_line_items(line_items):
+    line_item_dicts = []
+    for line_item in line_items:
+        line_item_dicts.append({
+            'title': line_item.item.title,
+            'seq': line_item.line_item_seq,
+            'id': line_item.id,
+            'category': line_item.item.category.title,
+            'description': line_item.item.description.text,
+            'currency': CURRENCY_SYMBOL['INR'],
+            'base_amount': line_item.base_amount,
+            'discounted_amount': line_item.discounted_amount,
+            'final_amount': line_item.final_amount,
+            'discount_policy': line_item.discount_policy.title if line_item.discount_policy else "",
+            'discount_coupon': line_item.discount_coupon.code if line_item.discount_coupon else "",
+            'cancelled_at': invoice_date_filter(line_item.cancelled_at, '%d %b %Y %H:%M:%S') if line_item.cancelled_at else "",
```

Comment:
We shouldn't have date formatting hardcoded like this.

1. It should be locale-specific, or
2. It should at least be a constant defined in a single place.